elb/elb.go

```diff
@@ -141,8 +150,13 @@ func (e *elb) attachToFrontEnds() error {
 	return nil
 }
 
-// FindFrontEndElbs finds all elbs tagged with 'sky.uk/KubernetesClusterFrontend=<labelValue>'
-func FindFrontEndElbs(awsElb ELB, labelValue string) (map[string]LoadBalancerDetails, error) {
+// FindFrontEndElbs supports finding elbs without Ingress Name for backwards compatibility
+func FindFrontEndElbs(awsElb ELB, frontendTagValue string) (map[string]LoadBalancerDetails, error) {
+	return FindFrontEndElbsWithIngressName(awsElb, frontendTagValue, "")
+}
```

Comment:
I thought the README specified this was a breaking change and that ingresses without tags would not attach to anything, but it seems we have backwards compatibility.

Reply:
This method is still required because feed-dns isn't being updated for multiple ingress controllers, hence it needs the full list.